### DISCUSSION TITLE
Macbook Air M1 2020 16GB RAM 512GB SSD vs Macbook Pro M1 2020 16GB RAM 512GB SSD

### DISCUSSION THREAD

**Discussion by SonPhung95** [community/community#146605] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

For a QA Automation Tester, Is MacBook Air M1 16GB/512GB eligible enough to run test scripts stably and efficiently in the long run or should we use the Macbook Pro with the same specs ? Anyone gives some advice ?

**Comment by cybersokari** [comment]
A Macbook Pro will be better because of Docker and Emulators.

**Comment by BjornFrancke** [comment]
The MacBook Pro is the better choice for long-term stability and efficiency, especially for running extensive test scripts and multitasking. The MacBook Air M1 may suffice for lighter testing needs but lacks the sustained performance under load due to its fanless design.

**Comment by Ravindulakmina** [comment]
I have been using a MacBook Pro for several years. and, I have also worked with my friends' MacBook Air laptops.My recommendation is a MacBook Pro, for many reasons.It suits you.
